src/grid.js

```diff
@@ -72,11 +88,15 @@ var Grid = function(table, options) {
     model: Column
   });
 
+  this.$height = options.height;
```

Comment:
I prefer a "size" attribute instead of two different attributes for width and height.